### PR TITLE
Theme color updates and message colors

### PR DESCRIPTION
**Pull request by mikewheaton**
The changes here look more extensive than they are – I mostly just changed the order of the message colors (info, success, alert, warning) and made sure that "orangeLighter" is moved in with the other accent colors.

The message colors have been an undocumented feature of Fabric. This pull request is just to tidy up the mixins and classes to prepare for documenting them. **There are no new classes or changes to the message colors.**

For the theme colors, `secondary`, `tertiary`, and `light` have all changed slightly.

### CODE REVIEW

**Comment by mikewheaton** [comment]
Message colors have been added to the documentation site.

**Comment by mikewheaton** [comment]
Updated the changed theme colors for the documentation site.

File: src/sass/_Fabric.Color.Variables.scss [hunk 1]
```diff
@@ -66,14 +67,13 @@ $ms-color-green:         #107c10;
...
 
-//== Other Colors
+//== Message colors
 //
-$ms-color-error:             $ms-color-redDark;
-$ms-color-errorBackground:   #fde7e9;
+$ms-color-info:              $ms-color-green;
+$ms-color-infoBackground:    $ms-color-neutralLighter;
 $ms-color-success:           $ms-color-green;
 $ms-color-successBackground: #dff6dd;
```
Comment (notandrew) [inline src/sass/_Fabric.Color.Variables.scss]: Should this be in a variable instead?

Comment (mikewheaton) [inline src/sass/_Fabric.Color.Variables.scss]: What should be in a variable?

Comment (notandrew) [inline src/sass/_Fabric.Color.Variables.scss]: The successBackground & errorBackground (not shown in the above code snippit)

Comment (mikewheaton) [inline src/sass/_Fabric.Color.Variables.scss]: Oh I see. Those actually are variables and are with the others in `_Fabric.Color.Variables.scss`. It's just that some of the other message color variables are mapped to existing colors.

**Comment by notandrew** [comment]
looks good

**Pull request merged**